Instrumentation test fails on API level 21 with exception: android.view.InflateException

**Describe the bug**
Instrumentation test fails on API level 21 with exception: android.view.InflateException

**Component**
SDC gallery app

**To Reproduce**
Steps to reproduce the behavior:
1. Launch Nexus 5(API level 21) emulator
2. Run Data Capture gallery instrumentation tests on Nexus 5 device
3. Observe report

**Expected behavior**
All tests should get passed without any exception

**Screenshots/ file** 
![Screenshot from 2021-12-15 15-08-23](https://user-images.githubusercontent.com/64417713/146161991-3de6a7aa-409b-42ed-885a-01d2d6a152c6.png)
![Screenshot from 2021-12-15 15-11-21](https://user-images.githubusercontent.com/64417713/146162340-8f0c9e5a-1327-4a58-966d-a13dc11a64fa.png)
![Screenshot from 2021-12-15 15-11-34](https://user-images.githubusercontent.com/64417713/146162350-bf5261cd-aadf-4b5d-bb03-320ca3745a2e.png)

**Smartphone (please complete the following information):**
 - Device: Nexus 5
 - Android version: 5.0.2
 - API level: 21(Google API)

**Stack trace:**

```
android.view.InflateException: Binary XML file line #100: Error inflating class com.google.android.material.textfield.MaterialAutoCompleteTextView
at android.view.LayoutInflater.createView(LayoutInflater.java:633)
at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:743)
at android.view.LayoutInflater.rInflate(LayoutInflater.java:806)
at android.view.LayoutInflater.inflate(LayoutInflater.java:504)
at android.view.LayoutInflater.inflate(LayoutInflater.java:414)
at com.google.android.fhir.datacapture.views.QuestionnaireItemViewHolderFactory.create$datacapture_debug(QuestionnaireItemViewHolderFactory.kt:36)
at com.google.android.fhir.datacapture.views.QuestionnaireItemAutoCompleteViewHolderFactoryInstrumentedTest.setUp(QuestionnaireItemAutoCompleteViewHolderFactoryInstrumentedTest.kt:56)
at java.lang.reflect.Method.invoke(Native Method)
at java.lang.reflect.Method.invoke(Method.java:372)
at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
at androidx.test.internal.runner.junit4.statement.RunBefores.evaluate(RunBefores.java:76)
at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
at androidx.test.ext.junit.runners.AndroidJUnit4.run(AndroidJUnit4.java:154)
at org.junit.runners.Suite.runChild(Suite.java:128)
at org.junit.runners.Suite.runChild(Suite.java:27)
at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
at org.junit.runner.JUnitCore.run(JUnitCore.java:115)
at androidx.test.internal.runner.TestExecutor.execute(TestExecutor.java:56)
at androidx.test.runner.AndroidJUnitRunner.onStart(AndroidJUnitRunner.java:388)
at android.app.Instrumentation$InstrumentationThread.run(Instrumentation.java:1853)
Caused by: java.lang.reflect.InvocationTargetException
at java.lang.reflect.Constructor.newInstance(Native Method)
at java.lang.reflect.Constructor.newInstance(Constructor.java:288)
at android.view.LayoutInflater.createView(LayoutInflater.java:607)
... 39 more
Caused by: java.lang.RuntimeException: Can't create handler inside thread that has not called Looper.prepare()
at android.os.Handler.<init>(Handler.java:200)
at android.os.Handler.<init>(Handler.java:114)
at android.widget.ListPopupWindow.<init>(ListPopupWindow.java:106)
at android.widget.AutoCompleteTextView.<init>(AutoCompleteTextView.java:143)
at android.widget.AutoCompleteTextView.<init>(AutoCompleteTextView.java:136)
at androidx.appcompat.widget.AppCompatAutoCompleteTextView.<init>(AppCompatAutoCompleteTextView.java:76)
at com.google.android.material.textfield.MaterialAutoCompleteTextView.<init>(MaterialAutoCompleteTextView.java:75)
at com.google.android.material.textfield.MaterialAutoCompleteTextView.<init>(MaterialAutoCompleteTextView.java:70)
... 42 more
```

--- a/datacapture/src/androidTest/java/com/google/android/fhir/datacapture/views/QuestionnaireItemAutoCompleteViewHolderFactoryInstrumentedTest.kt
+++ b/datacapture/src/androidTest/java/com/google/android/fhir/datacapture/views/QuestionnaireItemAutoCompleteViewHolderFactoryInstrumentedTest.kt
@@ -45,6 +45,7 @@ class QuestionnaireItemAutoCompleteViewHolderFactoryInstrumentedTest {
   private lateinit var viewHolder: QuestionnaireItemViewHolder
 
   @Before
+  @UiThreadTest
   fun setUp() {
     context =
       ContextThemeWrapper(

--- a/datacapture/src/androidTest/java/com/google/android/fhir/datacapture/views/QuestionnaireItemDropDownViewHolderFactoryInstrumentedTest.kt
+++ b/datacapture/src/androidTest/java/com/google/android/fhir/datacapture/views/QuestionnaireItemDropDownViewHolderFactoryInstrumentedTest.kt
@@ -44,6 +44,7 @@ class QuestionnaireItemDropDownViewHolderFactoryInstrumentedTest {
   private lateinit var viewHolder: QuestionnaireItemViewHolder
 
   @Before
+  @UiThreadTest
   fun setUp() {
     context =
       ContextThemeWrapper(

